.github/workflows/ci.yaml

```diff
@@ -34,10 +40,10 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: true
-      - name: Run Coverage
+      - name: +run-coverage on Earthly satellite
```

Comment:
Nit: You've been inconsistent about the `Run ` prefix on these, FWIW I think it reads better with it.

Reply:
I'm going to raise a follow up PR to make `earthly docs` work and will add the `Run` back there.  I was probably overthinking it when I was trying to avoid `Run +run-...`.  I like consistency too.